tasks/db_deploy/test/manual_tests/README.md

```diff
@@ -764,36 +781,26 @@ Type "help" for help.
 postgres=# \c disaster_recovery
 You are now connected to database "disaster_recovery" as user "postgres".
 
-disaster_recovery=# \i sql/orca_schema_v2/remove.sql
+disaster_recovery=# \i sql/orca_schema_v4/remove.sql
 
-psql:sql/orca_schema_v2/remove.sql:1: NOTICE:  drop cascades to 4 other objects
-DETAIL:  drop cascades to table orca.schema_versions
-drop cascades to table orca.recovery_status
-drop cascades to table orca.recovery_job
-drop cascades to table orca.recovery_file
-DROP SCHEMA
-DROP ROLE
-REVOKE
-REVOKE
-DROP ROLE
-REVOKE
-DROP ROLE
-psql:sql/orca_schema_v2/remove.sql:8: WARNING:  there is no transaction in progress
+psql:sql/orca_schema_v4/remove.sql:1: NOTICE:  drop the 5 inventory tables
```

Comment:
This output is not correct. The output you would see would be something like
```
DROP TABLE 
DROP TABLE 
DROP TABLE 
DROP TABLE 
DROP TABLE 
```

Reply:
added

Reply:
Remove `psql:sql/orca_schema_v4/remove.sql:1: NOTICE:  drop the 5 inventory tables` line